Explorer/Assets/DCL/SDKComponents/Tween/Components/CustomTweener/CustomTweenerImpl.cs

```diff
@@ -88,6 +88,32 @@ public override void UpdateTransform(Transform transform)
         {
             transform.localRotation = currentValue;
         }
+    }
+
+    public class TextureMoveTweener : CustomTweener<Vector2, VectorOptions>
```

Comment:
That's not entirely possible, since we still depend on the PBTween mode (position, scale, rotation, textureMove) to create the custom tweener and get the values.
We might add another abstraction layer but I'm not sure it's worth it